### PR TITLE
Bump k6 version

### PR DESCRIPTION
**Pull request by codebien**
Fixes https://github.com/grafana/xk6-output-influxdb/issues/27

### CODE REVIEW

**Comment by diversario** [comment]
This helped compile influx + promrw in the same binary. Would be great to see this land!

**Comment by codebien** [comment]
@joanlopez @mstoykov it requires to get https://github.com/grafana/xk6-output-influxdb/pull/29 merged before.

File: go.mod [hunk 1]
```diff
@@ -5,32 +5,47 @@ go 1.19
 require (
 	github.com/influxdata/influxdb-client-go/v2 v2.12.2
 	github.com/mstoykov/envconfig v1.4.1-0.20220114105314-765c6d8c76f1
-	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.2
-	go.k6.io/k6 v0.45.1
-	gopkg.in/guregu/null.v3 v3.5.0
```
Comment (joanlopez) [inline go.mod:11]: Nit; this downgrade caught my attention, but I've seen that `v3.3.0` is the same version that k6 uses, so I guess it's fine/intended.

**Review by joanlopez — APPROVED** [review]

File: go.mod (hunk 1 above)
Comment (codebien) [reply go.mod:11]: Yes, it is what `go-depsync` suggested.

**Review by mstoykov — APPROVED** [review]

**Pull request merged**